src/assets/Main.php

```diff
@@ -27,6 +27,7 @@ class Main extends \luya\web\Asset
      */
     public $js = [
         'dist/js/main.js',
+        'dist/js/echarts.min.js',
```

Comment:
echarts must be provided as bower-asset library: => https://github.com/luyadev/luya-module-admin/blob/master/composer.json#L26 

then add like this to gulp process: 
https://github.com/luyadev/luya-module-admin/blob/master/src/resources/compileConfig.js#L78